### PR TITLE
Add parameter for inbox-reader-delay

### PR DESCRIPTION
**Pull request by joshuacolvin0**

### CODE REVIEW

File: packages/arb-util/configuration/configuration.go [hunk 1]
```diff
@@ -194,19 +194,23 @@ type Forwarder struct {
 	RpcMode   string `koanf:"rpc-mode"`
 }
 
+type InboxReader struct {
+	DelayBlocks int64 `koanf:"delay-blocks"`
+	Paranoid    bool  `koanf:"paranoid-inbox-reader"`
```
Comment (hkalodner) [inline packages/arb-util/configuration/configuration.go:199]: This should just be `paranoid`

**Review by hkalodner — CHANGES_REQUESTED** [review]
Looks good other than comment

File: packages/arb-util/configuration/configuration.go (hunk 1 above)
Comment (joshuacolvin0) [reply packages/arb-util/configuration/configuration.go]: good catch

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**